docs/UsersGuide/source/BuildingRunningTesting/BuildSRW.rst

```diff
@@ -198,7 +190,7 @@ On Level 1 systems for which a modulefile is provided under the ``modulefiles``
 
    ./devbuild.sh --platform=<machine_name>
 
-where ``<machine_name>`` is replaced with the name of the platform the user is working on. Valid values include: ``cheyenne`` | ``gaea`` | ``hera`` | ``jet`` | ``linux`` | ``macos`` | ``noaacloud`` | ``orion`` 
+where ``<machine_name>`` is replaced with the name of the platform the user is working on. Valid values include: ``derecho`` | ``gaea`` | ``hera`` | ``hercules`` | ``jet`` | ``linux`` | ``macos`` | ``noaacloud`` | ``orion``
```

Comment:
@gspetro-NOAA In the file `ConfigWorkflow.rst` below, I saw "GAEA-C5" as a valid option for `MACHINE` (in addition to plain-old "GAEA") but I don't see it in this list.  Is that all consistent?

Reply:
I didn't update because it says valid values "include," which suggests there may be more, and I was afraid listing two gaeas would confuse newbies. I also don't want to promise that every list of machines throughout the documentation is comprehensive (maintainability issue), but I do think it's typically more convenient for users (especially newer ones) to have a list of most machines in front of them (instead of having to click around to other parts of the documentation). Ultimately, I want `ConfigWorkflow.rst` to have the definitive list, so maybe I should just link there instead (which I often do), but the clicking around can be confusing if you just want to run from start to finish. In short, trying to balance a bunch of concerns! I think I'll just change it to a crosslink to the complete list to save my sanity/increase maintainability.